src/model_api.jl

```diff
@@ -182,3 +184,55 @@ If for some reason a model is sometimes unable to report feature importances the
 
 """
 function feature_importances end
+
+_named_tuple(named_tuple::NamedTuple) = named_tuple
+_named_tuple(::Nothing) = NamedTuple()
+_named_tuple(something_else) = (report=something_else,)
+_scrub(x) = x
+_scrub(x::NamedTuple) = isempty(x) ? nothing : x
+_keys(named_tuple) = keys(named_tuple)
+_keys(::Nothing) = ()
+
+"""
+    MLJModelInterface.report(model, report_given_method)
+
+Merge the reports in the dictionary `report_given_method` into a single
+property-accessible object. It is supposed that the keys of the dictionary are
+`:fit` and the symbolic names of MLJModelInterface.jl operations, such as `:predict` or
```

Comment:
```suggestion
property-accessible object. It is supposed that the keys of the dictionary consist of 
`:fit` and optionally one or more symbolic names of MLJModelInterface.jl operations, such as `:predict` or
```
I'm thinking that `fit` must always be contained in the dictionary, right?

Reply:
Good question. 

I think that in the current proposal at MLJBase, there will always be a `:fit` key because even `Static` machines get `fit!` (at first operation call). However, it's possible this detail might change in the future, so I'd not like to enforce that. But the docstring reads poorly and I have tweaked it.

Reply:
The docstring looks much better.